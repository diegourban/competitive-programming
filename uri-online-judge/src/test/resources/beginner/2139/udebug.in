12 24
12 25
1 30
2 29
3 15
4 1
7 3
9 30
10 21
5 31
12 30
6 5