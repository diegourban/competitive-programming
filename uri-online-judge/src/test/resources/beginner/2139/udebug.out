E vespera de natal!
E natal!
Faltam 330 dias para o natal!
Faltam 300 dias para o natal!
Faltam 285 dias para o natal!
Faltam 268 dias para o natal!
Faltam 175 dias para o natal!
Faltam 86 dias para o natal!
Faltam 65 dias para o natal!
Faltam 208 dias para o natal!
Ja passou!
Faltam 203 dias para o natal!
